### DISCUSSION TITLE
second-order phase transition calculations

### DISCUSSION THREAD

**Discussion by bcai0411** [SSCHAcode/python-sscha#366] [Q&A]
I am doing a new test of second-order phase transition calculation. But the code said "Not all or too many NN assigned 12" when I run "ff_claculator = ff.Calculator.ToyModelCalculator(ff_dyn)". Does anyone know how to solve it?

The example of SnTe_Toymodel was done successfully. And I have checked the dyn files. It seems all dyn files contain the same information. So I am confused.

Thanks in advances.
[NewTest.zip](https://github.com/user-attachments/files/18047681/NewTest.zip)

**Comment by mesonepigreco** [comment]
Hi, can you specify the full error stack? we need it to reconstruct exactly the tree of functions call that is raising the issue
